src/channel.h

```diff
@@ -98,91 +94,134 @@ class CChannel : public QObject
     bool GetAddress ( CHostAddress& RetAddr );
     const CHostAddress& GetAddress() const { return InetAddr; }
 
-    void ResetInfo() { ChannelInfo = CChannelCoreInfo(); } // reset does not emit a message
-    QString GetName();
-    void SetChanInfo ( const CChannelCoreInfo& NChanInf );
+    void ResetInfo()
+    {
+        ChannelInfo = CChannelCoreInfo();
+    } // reset does not emit a message
+    QString           GetName();
+    void              SetChanInfo ( const CChannelCoreInfo& NChanInf );
     CChannelCoreInfo& GetChanInfo() { return ChannelInfo; }
 
     void SetRemoteInfo ( const CChannelCoreInfo ChInfo )
-        { Protocol.CreateChanInfoMes ( ChInfo ); }
+    {
```

Comment:
Ah!  Here we go.  An example of how a wide header method gets wrapped - body remains single line.

Reply:
it can't do that. only keeps it together if the whole header method fits into a single line

Reply:
... and I think this is the best way to do, so we could mark this as resolved?